quickwit-metastore/src/metastore/single_file_metastore.rs

```diff
@@ -289,8 +289,7 @@ impl Metastore for SingleFileMetastore {
         let metadata_set = self.get_index(index_id).await?;
         let splits = metadata_set
             .splits
-            .into_iter()
-            .map(|(_, split_metadata)| split_metadata)
+            .into_values()
```

Comment:
Let me guess: this feature is stabilized in rust 1.54 but CI uses an older version?

Reply:
Yeah, I need to publish a new version of the builder. Actually, I'm going to automate it.